Windows 10 node-gyp rebuild failing

I can't get torus to install on windows 10. I've tried completely uninstall node, tried version `6.11.4 AND 8.7.0`, I've installed the `windows-build-tools`. I keep getting this error. Any ideas??

```
> kexec@3.0.0 install C:\Users\webbr\AppData\Roaming\npm\node_modules\torus-cli\node_modules\kexec
> node-gyp rebuild


C:\Users\webbr\AppData\Roaming\npm\node_modules\torus-cli\node_modules\kexec>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
The system cannot find the path specified.
gyp: Call to 'node -v |grep "v[^0]" > /dev/null && echo "__NODE_GE_V4__" || true' returned exit status 255 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\configure.js:305:16)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:219:12)
gyp ERR! System Windows_NT 10.0.15063
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\webbr\AppData\Roaming\npm\node_modules\torus-cli\node_modules\kexec
gyp ERR! node -v v6.11.4
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "torus-cli" "-g"
npm ERR! node v6.11.4
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE

npm ERR! kexec@3.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the kexec@3.0.0 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the kexec package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs kexec
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls kexec
npm ERR! There is likely additional logging output above.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,10 @@
 
 _Unreleased_
 
+**Fixes**
+
+- Fixed a bug preventing Torus from being used once installed via npm on win32.
+
 ## v0.25.1
 
 _2017-10-16_

--- a/Makefile
+++ b/Makefile
@@ -424,7 +424,9 @@ NPM_DEPS=\
 	builds/npm/LICENSE.md \
 	builds/npm/bin/torus \
 	builds/npm/bin/torus-darwin-amd64 \
-	builds/npm/bin/torus-linux-amd64
+	builds/npm/bin/torus-linux-amd64 \
+	builds/npm/bin/torus.exe
+
 npm: $(NPM_DEPS)
 
 builds/npm builds/npm/bin builds/npm/scripts:
@@ -443,6 +445,9 @@ builds/npm/bin/torus-darwin-amd64: builds/bin/$(VERSION)/darwin/amd64/torus buil
 	cp $< $@
 
 builds/npm/bin/torus-linux-amd64: builds/bin/$(VERSION)/linux/amd64/torus builds/npm/bin
+	cp $< $@
+
+builds/npm/bin/torus.exe: builds/bin/$(VERSION)/windows/amd64/torus.exe builds/npm/bin
 	cp $< $@
 
 builds/torus-npm-$(VERSION).tar.gz: npm

--- a/packaging/npm/package.json.in
+++ b/packaging/npm/package.json.in
@@ -33,9 +33,9 @@
   ],
   "logo": "https://s3.amazonaws.com/manifold.co/torus_logo.png",
   "license": "BSD-3-Clause",
-  "dependencies": {
+  "dependencies": {},
+  "devDependencies": {},
+  "optionalDependencies": {
     "kexec": "^3.0.0"
-  },
-  "devDependencies": {
   }
 }

--- a/packaging/npm/passthrough.js
+++ b/packaging/npm/passthrough.js
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
 var path = require('path');
-var kexec = require('kexec');
 
 // os and arch restrictions are handled by the package.json
 var os = process.platform;
@@ -10,4 +9,27 @@ var arch = 'amd64';
 // arguments. This is a true exec(3), which will take over the pid, env, and
 // file descriptors.
 var torusPath = path.join(__dirname, '../bin/torus-' + os + '-' + arch);
-kexec(torusPath, process.argv.slice(2));
+if (os === 'win32') {
+  torusPath = path.join(__dirname, '../bin/torus.exe');
+}
+
+try {
+  var kexec = require('kexec');
+  kexec(torusPath, process.argv.slice(2));
+} catch (err) {
+  if (err.code !== 'MODULE_NOT_FOUND') {
+    console.error('Could not leverage kexec due to error: ' + err.message);
+  }
+
+  var spawn = require('child_process').spawn;
+  var proc = spawn(torusPath, process.argv.slice(2), { stdio: 'inherit' });
+  proc.on('exit', function (code, signal) {
+    process.on('exit', function () {
+      if (signal) {
+        process.kill(process.pid, signal);
+      } else {
+        process.exit(code);
+      }
+    });
+  });
+}

